src/integrationTest/java/uk/gov/hmcts/reform/laubackend/idam/bdd/UserDeletionAuditGetSteps.java

```diff
@@ -74,5 +74,15 @@ public void responseIsReturned(int statusCode, String status) {
 
     }
 
+    @Then("multiple deleted account entries are returned with size {int}")
+    public void multipleAccountEntriesReturnedWithCount(Integer count)
+        throws InvocationTargetException, IllegalAccessException, NoSuchMethodException {
```

Comment:
The method does not trow this exceptions

Reply:
This has been removed